enterprise/frontend/src/embedding-sdk/components/public/CollectionBrowser/CollectionBrowser.tsx

```diff
@@ -37,6 +37,7 @@ type CollectionBrowserProps = {
   onClick?: (item: CollectionItem) => void;
   pageSize?: number;
   visibleEntityTypes?: UserFacingEntityName[];
+  EmptyContentComponent?: (() => JSX.Element) | null;
```

Comment:
Wondering if `() => JSX.Element` should be `React.FC` instead

Reply:
That was the one! I couldn't think of the type so I went with this one. I'll change this now